src/watson/backends.py

```diff
@@ -174,10 +169,13 @@ class PostgresSearchBackend(SearchBackend):
 
     def escape_postgres_query(self, text):
         """Escapes the given text to become a valid ts_query."""
+        # PostgreSQL to_tsquery operators: ! & : ( ) |
+        chars_to_escape = re.compile(r'[&:(|)!]', re.UNICODE)
```

Comment:
This regex is repeated twice in both postgres backends. We should move it to a global constant `RE_POSTGRES_ESCAPE_CHARS`. That way we also benefit from only compiling the regex once.